py3status/modules/wifi.py

```diff
@@ -0,0 +1,84 @@
+"""
+Display the current wifi ESSID and IP address.
+It improves on the default i3status wifi module by allowing the user
+to toggle between the ESSID and IP address with a mouse click.
+
+This module is inspired by the "whatismyip" module.
+
+Requires:
+    - iwlib (https://pypi.python.org/pypi/iwlib)
+
+Configuration parameters:
+    - mode: default mode to display. Can be 'essid' or 'ip'
+    - interface: the wireless interface to query. (default wlan0)
+
+@author Tjaart van der Walt (github:tjaartvdwalt)
+@license BSD
+"""
+
+import iwlib
+import socket
+from time import time
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+    cache_timeout = 30
```

Comment:
I will fix this in my next commit. 
Is there any guide on what the default `cache_timeout` value should be?
Looking at other modules it seems to range anywhere from 5-30.